packages/e2e/README.md

```diff
@@ -54,11 +59,22 @@ The configuration provided by this package is kept very minimal:
 }
 ```
 
-For the remaining options, set by _cypress_ itself, please refer to its documentation: https://docs.cypress.io/guides/references/configuration
+If you want to overwrite this, you can do that in the `.factorialrc.js`:
+
+```js
+// .factorialrc.js
+
+module.exports = {
+  use: [require("@factorial/stack-e2e"), {
+    … // config object
+  }],
+};
+```
 
-## Extending or overwriting configuration
+If you want to overwrite the normal configuration, you have two options:
 
-You can create a `cypress.json` (this is the standard by _cypress_) in your root folder, which will then be merged with our [default configuration](#default-configuration).
```

Comment:
is that link still working when headline has been changed above?

Reply:
Ah, good catch, thanks!